components/Misc/Alert.module.scss

```diff
@@ -1,28 +1,27 @@
-@import "../../styles/globals.scss";
+@use "../../styles/globals.scss" as g;
 
 .alert {
-    @include rounded("sm");
-    width: 100%;
-    border-width: 1px;
-    display: flex;
-    flex-direction: row;
-    justify-content: center;
-    align-items: center;
-    padding: 5px;
-    margin: 15px;
-    font-size: 18px;
+  @include g.rounded("md");
+  @include g.flex-row;
+  width: 100%;
+  padding: 0.3rem;
+  margin: 0.9rem;
+  font: {
+      size: 1.1rem;
+      weight: 700;
+  }
 }
 
 .alert-error {
-    background-color: rgba($color: red, $alpha: .3);
-    border: 1px solid red;
+  background-color: rgba($color: red, $alpha: 0.6);
```

Comment:
Why did you change the opacity from 0.3 to 0.6?

Reply:
It makes the alert look more like an alert.

Reply:
alrighty